ansible/config_sonic_basedon_testbed.yml

```diff
@@ -543,6 +543,13 @@
       - name: execute cli "config load_minigraph --override_config -y" to apply new minigraph
         become: true
         shell: config load_minigraph --override_config -y
+        register: load_minigraph_rc
+        ignore_errors: yes
+
+      - name: execute cli "config load_minigraph -y" to apply new minigraph
+        become: true
+        shell: config load_minigraph -y
+        when: load_minigraph_rc.rc != 0
```

Comment:
It may be a better idea to do `stderr` check and look for the regex/string in it. That way we tightly control this change and limit it to what is needed.